andela/hc-rogue-one
# Ft 156577849 set user priorities

#### What does this PR do?
This PR ensures that User should be able to set up a list of users(priority list) to be notified one after the other when a job fails until the alert is resolved
#### Description of Task to be completed?
A team access account super-user should be able to define who in the directory of the with team access should be notified about a failed job, when they should be notified and in what order

#### How should this be manually tested?
1. Clone the application and install the requirements in the requirements.txt
2.Run the application and click on account settings
3: Add a team member.
4. Set their priority for each check
#### What are the relevant pivotal tracker stories?
#156577849

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

- [] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
